src/preprocessing-service/preprocess.py

```diff
@@ -0,0 +1,95 @@
+# Standard Library
+import asyncio
+import logging
+
+# Third Party
+import pandas as pd
+from elasticsearch import AsyncElasticsearch
+from elasticsearch.helpers import async_streaming_bulk
+from masker import LogMasker
+from nats_wrapper import NatsWrapper
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(name)s - %(message)s")
+
+ES_ENDPOINT = "https://opendistro-es-client-service.default.svc.cluster.local:9200"
```

Comment:
let's make this an ENV variable? so it can be specified in the yaml file.

Reply:
Yes, agreed add to the yaml file.

Reply:
added